taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/impl/OutboxRestServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package pro.taskana.adapter.camunda.outbox.rest.impl;
+
+import com.google.gson.Gson;
+import pro.taskana.adapter.camunda.outbox.rest.OutboxRestService;
+import pro.taskana.adapter.camunda.outbox.rest.dto.ReferencedTaskDTO;
+
+import javax.naming.InitialContext;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class OutboxRestServiceImpl implements OutboxRestService {
+
+    private final Logger LOGGER = Logger.getLogger(this.getClass().getName());
+
+    public List<ReferencedTaskDTO> getCreateEvents() {
+
+        List<ReferencedTaskDTO> referencedTaskDTOS = new ArrayList<ReferencedTaskDTO>();
+
+        try {
+
```

Comment:
if the try block is left via an exception, the connection is not closed.
Proposal: use try-with-resources syntax:
try(Connection connection = getConnection()( {
...
connection.commit;
} catch ....